src/StockportGovUK.NetStandard.Gateways.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <Title>A package that provides base gateways using HttpClient</Title>
-    <VersionPrefix>6.0.2</VersionPrefix>
+    <VersionPrefix>6.1.1</VersionPrefix>
```

Comment:
You have bumped both the minor and the patch now, we need to bump just the patch so the version would be 6.0.3

Reply:
Sorry, updated the version to the correct one